### DISCUSSION TITLE
Install script stops after selecting USB RFID reader

### DISCUSSION THREAD

**Discussion by maxbobse** [MiczFlor/RPi-Jukebox-RFID#1635] [Q&A] [answered]
Hi,

I am running the one line install script on a Raspi 3 under 11 bullseye. When I get to the point where I have to select the RFID reader from the list of options

```
If you are using an RFID reader, connect it to your RPi.
(In case your RFID reader required soldering, consult the manual.)
Have you connected your RFID reader? [Y/n] y
Please select the RFID reader you want to use
1) USB-Reader (e.g. Neuftech)
2) RC522
3) PN532
4) Manual configuration
5) Multiple RFID reader
#? 1
Choose the reader from list
0 vc4
1 Sycreader RFID Technology Co., Ltd SYC ID&IC USB Reader
2 C-Media Electronics Inc.       USB PnP Sound Device
Device Number: 1
```

I enter 1 and confirm with Enter and afterwards nothing happens.
Any ideas why this isn't working?

Max

**Comment by Thilo-90** [comment, marked as answer]
Hi @maxbobse,
have you tried exiting this selection with  Ctrl+C? 
That worked for me when setting up my reader (different topic though). I did not get any confirmation on console screen.
I just tried the Reader in the Webapplication afterward.

**Reply by maxbobse** [reply]
Hey @Thilo-90 
Thanks, that actually worked! 👍
